src/data_sets.h

```diff
@@ -1467,6 +1467,11 @@ enum eGenFaultCodes
     GFC_CHIP_ERASE_INVALID				= 0x01000000,
     /*! Fault: GPS time fault */
     GFC_GNSS_TIME_FAULT                 = 0x02000000,
+    /*! Fault: GPS receiver time fault */
+    GFC_GNSS_RECEIVER_TIME              = 0x04000000,
```

Comment:
This feels a little redundant.  It probably worth clarifying the differences in GNSS_RECEIVER_TIME and GNSS_TIME_FAULT in the comments.  Each are GNSS, TIME, and FAULTs.  The assumption is that one it not a receiver fault, but the other is; but what does that really mean?